### PR TITLE
Added Joshua Keel's personal site (joshuakeel.com).

### PR DESCRIPTION
**Pull request by joshuakeel**
Thanks, Andy! Love the site. :)

### CODE REVIEW

**Comment by hankchizljaw** [comment]
Welcome!

File: sites/joshuakeel.com.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: 'Joshua Keel'
+url: 'https://joshuakeel.com'
+tags: ['writer', 'web developer', 'philosophy', 'ethics', 'godless heathen']
```
Comment (hankchizljaw) [inline sites/joshuakeel.com.md]: > godless heathen

go to every-layout.dev and view source ;)

Comment (joshuakeel) [reply sites/joshuakeel.com.md]: Oooh, nice easter egg. ;)

Comment (hankchizljaw) [reply sites/joshuakeel.com.md]: 😎

**Pull request merged**